setup.sh

```diff
@@ -202,10 +204,9 @@ apt -y install python3-doc
 
 # CPP Reference
 
-wget -O /tmp/html_book.zip https://github.com/PeterFeicht/cppreference-doc/releases/download/v20220730/html-book-20220730.zip
+$wget https://github.com/PeterFeicht/cppreference-doc/releases/download/v20220730/html-book-20220730.zip
 mkdir -p /usr/share/doc/cppreference
-unzip -o /tmp/html_book.zip -d /usr/share/doc/cppreference
-rm -f /tmp/html_book.zip
+unzip -o $cache/html-book-20220730.zip -d /usr/share/doc/cppreference
```

Comment:
There is a duplication of version here. It was probably intended to download to a fix location above:

```bash
wget -O /tmp/html_book.zip https://github.com/PeterFeicht/cppreference-doc/releases/download/v20220730/html-book-20220730.zip
```

Reply:
The wget command I use will not download the file if it already exists. That's why I always include a version number in the local filename as well. (otherwise if the url changed but the cache file stays then it won't download the new version)

Perhaps there's a better way to do this but this is all I managed to come up with.

Reply:
I think reentrancy (allowing `setup.sh` to run multiple times) would be a nice-to-have feature, simply for debugging.

Reply:
Meanwhile, I understand the idea behind this change.

Reply:
Actually... Yes, I've been thinking that it would be an upgrade. Either ansible or cloud-init. (latter already set up and supported, but not required)